apps/main/admin.py

```diff
@@ -56,8 +56,8 @@ def get_urls(self):
     def multiple_upload_files(self, request):
         form = FileFieldForm()
         user = request.user
-        can_add_file = user.has_perm('core.add_uploadfile')
-        if not can_add_file:
+        permission = user.get_group_permissions()
+        if not('main.add_uploadedfile' in permission):
```

Comment:
Parentheses are redundant.
```suggestion
        if not 'main.add_uploadedfile' in permission:
```